src/MPCounters.jl

```diff
@@ -1,4 +1,4 @@
-export MPCounters, increment!, reset!, decrement!
+export MPCounters, reset!, decrement!
```

Comment:
I would suggest avoid re-exporting these, then.
```suggestion
export MPCounters
```